api/api.yaml

```diff
@@ -165,16 +165,16 @@ definitions:
           $ref: '#/definitions/SearchResult'
   SearchResult:
     description: >
-      A field. For example, a BigQuery Gender column could include the field
-      name "Gender", description "Sex at birth" and elasticsearch field name
-      "project_id.dataset_id.table_name.Gender".
+      Can be a field (Gender), or a field and selected value (Gender = female).
```

Comment:
Sorry I missed this in last PR. On line 162, fields -> search_results

Reply:
Done